sympy/parsing/autolev/__init__.py

```diff
@@ -1,56 +1,60 @@
+
+    Returns
+    =======
+    sympy_code : str
+        Equivalent sympy and/or numpy/pydy code as the input code.
 
-    output: str
-            1. Can be the name of an output file to which the SymPy code should be written to.
-            2. Can be the string "print". In this case the SymPy code is written to stdout.
-            3. Can be the string "list". In this it returns a list containing the SymPy code.
-                Each element in the list corresponds to one line of code.
 
     Example (Double Pendulum)
-    -------
+    =========================
+    Suppose this is the file we need to parse:
+    ------------FILE--------------
+    % double_pendulum.al
+    MOTIONVARIABLES' Q{2}', U{2}'
+    CONSTANTS L,M,G
+    NEWTONIAN N
+    FRAMES A,B
+    SIMPROT(N, A, 3, Q1)
+    SIMPROT(N, B, 3, Q2)
+    W_A_N>=U1*N3>
+    W_B_N>=U2*N3>
+    POINT O
+    PARTICLES P,R
+    P_O_P> = L*A1>
+    P_P_R> = L*B1>
+    V_O_N> = 0>
+    V2PTS(N, A, O, P)
+    V2PTS(N, B, P, R)
+    MASS P=M, R=M
+    Q1' = U1
+    Q2' = U2
+    GRAVITY(G*N1>)
+    ZERO = FR() + FRSTAR()
+    KANE()
+    INPUT M=1,G=9.81,L=1
+    INPUT Q1=.1,Q2=.2,U1=0,U2=0
+    INPUT TFINAL=10, INTEGSTP=.01
+    CODE DYNAMICS() some_filename.c
+    ------------FILE--------------
     >>> from sympy.parsing.autolev import parse_autolev  # doctest: +SKIP
-    >>> l = []  # doctest: +SKIP
-    >>> l.append("MOTIONVARIABLES' Q{2}', U{2}')"  # doctest: +SKIP
-    >>> l.append("CONSTANTS L,M,G)"  # doctest: +SKIP
-    >>> l.append("NEWTONIAN N)"  # doctest: +SKIP
-    >>> l.append("FRAMES A,B)"  # doctest: +SKIP
-    >>> l.append("SIMPROT(N, A, 3, Q1))"  # doctest: +SKIP
-    >>> l.append("SIMPROT(N, B, 3, Q2))"  # doctest: +SKIP
-    >>> l.append("W_A_N>=U1*N3>)"  # doctest: +SKIP
-    >>> l.append("W_B_N>=U2*N3>)"  # doctest: +SKIP
-    >>> l.append("POINT O)"  # doctest: +SKIP
-    >>> l.append("PARTICLES P,R)"  # doctest: +SKIP
-    >>> l.append("P_O_P> = L*A1>)"  # doctest: +SKIP
-    >>> l.append("P_P_R> = L*B1>)"  # doctest: +SKIP
-    >>> l.append("V_O_N> = 0>)"  # doctest: +SKIP
-    >>> l.append("V2PTS(N, A, O, P))"  # doctest: +SKIP
-    >>> l.append("V2PTS(N, B, P, R))"  # doctest: +SKIP
-    >>> l.append("MASS P=M, R=M)"  # doctest: +SKIP
-    >>> l.append("Q1' = U1)"  # doctest: +SKIP
-    >>> l.append("Q2' = U2)"  # doctest: +SKIP
-    >>> l.append("GRAVITY(G*N1>))"  # doctest: +SKIP
-    >>> l.append("ZERO = FR() + FRSTAR())"  # doctest: +SKIP
-    >>> l.append("KANE())"  # doctest: +SKIP
-    >>> l.append("INPUT M=1,G=9.81,L=1)"  # doctest: +SKIP
-    >>> l.append("INPUT Q1=.1,Q2=.2,U1=0,U2=0)"  # doctest: +SKIP
-    >>> l.append("INPUT TFINAL=10, INTEGSTP=.01)"  # doctest: +SKIP
-    >>> l.append("CODE DYNAMICS() double_pendulum.c)"  # doctest: +SKIP
-    >>> parse_autolev("\\n".join(l), "print")  # doctest: +SKIP
+    >>> print(parse_autolev(open("double_pendulum.al"), include_numeric=True))  # doctest: +SKIP
```

Comment:
Should use:
```python
with open("double_pendulum.al", 'r') as f:
    parse_autolev(f, include_numeric=True)
```

Reply:
Don't skip the doctest if possible.

Load the text like:

```pycon
>>> my_al_text = """\
stuff
more stuff
another thing"""
```

Reply:
I have changed the doctest and the test passes locally but Travis won't pass it because of some antlr import stuff.

Reply:
Does antlr need to be added to the `.travis.yml` file?

Reply:
Is it because antlr is not installed in the doc tests section on travis?

Reply:
Open an issue for this "antlr not available for doctests".

Reply:
Use the `doctests_depends_on` decorator to skip the doctest if a module isn't installed.